libs/map/src/actions/select/select-action.ts

```diff
@@ -53,6 +54,7 @@ export class VlSelectAction extends VlBaseMapAction {
             condition: click,
             style,
             layers,
+            multi: true,
```

Comment:
Kan het kwaad dat deze altijd op true staat of kunnen we deze beter enkel voor de multiple select op true zetten?

Reply:
Er was code aanwezig om bij het klikken op overlappende features telkens de volgende feature te selecteren.
Maar als multi op false staat zal deze code maar 1 feature krijgen en kon je dus nooit onderliggende features selecteren. Omdat de code er wel was om door de features te loopen vermoed ik dat het wel de bedoeling is dat dit zo werkt. Daarom dat ik die op true gezet heb ook al is het geen multiple select.

Reply:
zoals besproken in de standup mag multi op true blijven staan.